projects/experimental/components/pin/pin.styles.less

```diff
@@ -0,0 +1,110 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+:host {
+    .transition(~'transform, width, height, font, border');
+    .clearbtn();
+    position: relative;
+    display: flex;
+    width: var(--t-size);
+    height: var(--t-size);
+    align-items: stretch;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    border-radius: 100%;
+    font: var(--tui-font-text-s);
+    font-weight: bold;
+    box-sizing: border-box;
+    color: var(--tui-accent-text);
+    background: var(--tui-accent);
+    box-shadow: 0 0 0 0.125rem var(--tui-elevation-01);
+    border: 0 solid var(--tui-accent);
+    transform: translate(-50%, -50%);
```

Comment:
Why we need this ? 🤔

Reply:
It is done this way so that when we position pin on a map with top/left – that place is at the center of the pin.